src/main/java/net/dblsaiko/illuminate/mixin/ClientPlayerEntityMixin.java

```diff
@@ -19,9 +19,8 @@ public abstract class ClientPlayerEntityMixin {
     @Inject(method = "isMainPlayer()Z", at = @At("HEAD"), cancellable = true)
     private void isMainPlayer(CallbackInfoReturnable<Boolean> cir) {
```

Comment:
This mixin doesn't actually seem to do anything, I suspect if the camera entity isn't the player, the player is rendered anyway